src/rill.h

```diff
@@ -117,27 +122,23 @@ const char * rill_store_file(const struct rill_store *store);
 unsigned rill_store_version(const struct rill_store *store);
 rill_ts_t rill_store_ts(const struct rill_store *store);
 size_t rill_store_quant(const struct rill_store *store);
-size_t rill_store_keys(const struct rill_store *store);
-size_t rill_store_vals(const struct rill_store *store);
+size_t rill_store_keys_count(const struct rill_store *store, enum rill_col column);
```

Comment:
We'll have to find a better for this function but I can't think of anything appropriate at the moment.

Reply:
How about index_len? as this is what this func basically does?